Documentation/CRDs/Shared-Filesystem/ceph-fs-subvolumegroup-crd.md

```diff
@@ -48,10 +52,14 @@ If any setting is unspecified, a suitable default will be used automatically.
 
 * `filesystemName`: The metadata name of the CephFilesystem CR where the subvolume group will be created.
 
-* `pinning`: To distribute load across MDS ranks in predictable and stable ways. Reference: https://docs.ceph.com/en/latest/cephfs/fs-volumes/#pinning-subvolumes-and-subvolume-groups.
-    * `distributed`: Range: <0, 1>, for disabling it set to 0
-    * `export`: Range: <0-256>, for disabling it set to -1
-    * `random`: Range: [0.0, 1.0], for disabling it set to 0.0
+* `size`: Size of the Ceph Filesystem subvolume group.
+
+* `dataPoolName`: The data pool name for the Ceph Filesystem subvolume group layout.
+
+* `pinning`: To distribute load across MDS ranks in predictable and stable ways. Reference: <https://docs.ceph.com/en/latest/cephfs/fs-volumes/#pinning-subvolumes-and-subvolume-groups>.
```

Comment:
Not related to your changes, but let's use a link instead of raw URL.
```suggestion
* `pinning`: To distribute load across MDS ranks in predictable and stable ways. See the Ceph doc for [Pinning subvolume groups](https://docs.ceph.com/en/latest/cephfs/fs-volumes/#pinning-subvolumes-and-subvolume-groups).
```